### PR TITLE
show-user and following

### PR DESCRIPTION
**Pull request by tuanhnt1712**

### CODE REVIEW

**Comment by tuanhnt1712** [comment]
ready

File: app/views/users/_profile.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <% end %>
+  <% if user_signed_in? %>
+    <%= render "follow_form" %>
+  <% end %>
+</div>
+<div class="sidebar">
+  <%= link_to t(".following"), following_user_path(@user) %>
+  <strong id="following"><%= @user.following.size %></strong>
+  <%= link_to t(".followers"), followers_user_path(@user) %>
+  <strong id="followers"><%= @user.followers.count %></strong>
```
Comment (huongnguyenmta) [inline app/views/users/_profile.html.erb]: using `size` instead of `count`

File: config/application.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@ module FramgiaProject
   class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
+    config.action_view.embed_authenticity_token_in_remote_forms = true
     # -- all .rb files in that directory are automatically loaded.
```
Comment (huongnguyenmta) [inline config/application.rb]: remove all comment in this file

File: app/controllers/relationships_controller.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    current_user.follow @user
+    respond_to do |format|
+      format.html{redirect_to @user}
+      format.js
+    end
+  end
+
+  def destroy
+    @user = Relationship.find_by(id: params[:id]).followed
+    current_user.unfollow @user
```
Comment (huongnguyenmta) [inline app/controllers/relationships_controller.rb:16]: add empty line under this line

File: app/controllers/relationships_controller.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :authenticate_user!, only: [:create, :destroy]
+
+  def create
+    @user = User.find_by id: params[:followed_id]
+    current_user.follow @user
```
Comment (huongnguyenmta) [inline app/controllers/relationships_controller.rb:6]: add empty line under this line

**Comment by huongnguyenmta** [comment]
commented

**Comment by tuanhnt1712** [comment]
ready

**Pull request merged**